config/constant.go

```diff
@@ -26,6 +28,7 @@ const (
 	DefaultBootstrapContainerMaxRetry             = 2
 	DefaultBootstrapMaxConcurrent                 = 5
 	DefaultMetricsZpoolName                       = ""
+	DefaultContainerRelocationTimeoutInHour       = 10
```

Comment:
why the timeout is so long?

Reply:
when we did manual move container by using `lxc move` the times it took depends on container size, 
with average 100-200mbps, so it will takes more time if we move stateful component likes kafka or elasticsearch which has more than 1TB disk.

Reply:
can we do minutes instead of hour, it's okay to put 600 here

Reply:
updated mas @giosakti